modules/ROOT/pages/promote-api-task.adoc

```diff
@@ -3,25 +3,19 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-A tight integration of Anypoint Platform components extends the use of environments to API management. In this procedure, you change the environment of an API.
+Before promoting an API instance to another environment, ensure that you have the `Manage APIs Configuration` permission.
```

Comment:
I don't think we change fonts for permissions.

Reply:
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-fontchoices